From 94f58bd180fa0100ef83682510672d1173745716 Mon Sep 17 00:00:00 2001
From: Christian Heusel <christian@heusel.eu>
Date: Tue, 6 May 2025 14:19:57 +0200
Subject: [PATCH] [hsa-runtime] Add missing cstdint include

This fixes the build with gcc 15 which is now released in major
distributions such as Fedora and Arch Linux.

Signed-off-by: Christian Heusel <christian@heusel.eu>
---
 runtime/hsa-runtime/core/inc/amd_elf_image.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/runtime/hsa-runtime/core/inc/amd_elf_image.hpp b/runtime/hsa-runtime/core/inc/amd_elf_image.hpp
index 177465c0..335e116b 100644
--- a/runtime/hsa-runtime/core/inc/amd_elf_image.hpp
+++ b/runtime/hsa-runtime/core/inc/amd_elf_image.hpp
@@ -47,6 +47,7 @@
 #include <sstream>
 #include <vector>
 #include <memory>
+#include <cstdint>
 
 namespace rocr {
 namespace amd {
-- 
2.49.0

